src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -185,5 +183,11 @@ private static string[] GetSemiColonEscapedArgs(IEnumerable<string> args)
 
             return array;
         }
+
+        /// <summary>
+        /// Adding double quotes around the property helps MSBuild arguments parser and avoid incorrect splits on ',' or ';'.
+        /// </summary>
+        private static string SurroundWithDoubleQuotes(string s)
```

Comment:
https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs#L441

we have `AddDoubleQuotes` as extension on our repo...but I think it's ok atm...also because will be superseded by new api, right?

Reply:
It will be yes.